src/task/SubscribeAndReceiveSubscribeAckTask.ts

```diff
@@ -69,6 +69,11 @@ export default class SubscribeAndReceiveSubscribeAckTask extends BaseTask {
 
     this.context.videoStreamIndex.subscribeFrameSent();
 
+    // See comment above `fixUpSubscriptionOrder`
+    const videoSubscriptions = this.context.browserBehavior.requiresUnifiedPlan()
```

Comment:
Do we still need to do this? I thought we no longer support planB?

Reply:
Safari 12 ayy